mypy.ini

```diff
@@ -134,26 +134,27 @@ files = fixtures/mypy-stubs,
         src/sentry/utils/appleconnect/,
         src/sentry/utils/arroyo.py,
         src/sentry/utils/assets.py,
-        src/sentry/utils/hashlib.py,
-        src/sentry/utils/suspect_resolutions/commit_correlation.py,
-        src/sentry/utils/suspect_resolutions/metric_correlation.py,
-        src/sentry/utils/suspect_resolutions/resolved_in_active_release.py,
         src/sentry/utils/avatar.py,
         src/sentry/utils/codecs.py,
         src/sentry/utils/committers.py,
         src/sentry/utils/cursors.py,
         src/sentry/utils/dates.py,
         src/sentry/utils/email/,
         src/sentry/utils/event_frames.py,
+        src/sentry/utils/hashlib.py,
         src/sentry/utils/http.py,
         src/sentry/utils/integrationdocs.py,
+        src/sentry/utils/json.py,
```

Comment:
New

Reply:
yeah I split the commits so you can see what changed specifically